### PR TITLE
feat: add mock services to cli for unit testing

### PR DESCRIPTION
**Pull request by barthuijgen**

### CODE REVIEW

File: packages/tools/kadena-cli/src/services/fs.serice.ts
Comment (jessevanmuijden) [inline packages/tools/kadena-cli/src/services/fs.serice.ts:1]: the filename is missing a letter `v`

**Review by nillo — APPROVED** [review]

File: packages/tools/kadena-cli/src/services/index.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { memoryFileSystemService } from './fs.memory.service.js';
+import type { IFileSystemService } from './fs.service.js';
+import { fileSystemService } from './fs.service.js';
+export {
+  getCallHistory,
+  serviceCalledWith as mockServiceCalledWith,
```
Comment (jessevanmuijden) [inline packages/tools/kadena-cli/src/services/index.ts:6]: is this still needed?

**Review by jessevanmuijden — APPROVED** [review]

File: packages/tools/kadena-cli/src/services/index.ts (hunk 1 above)
Comment (barthuijgen) [reply packages/tools/kadena-cli/src/services/index.ts:6]: Looks like this `as` was typescript trying to help me in refactoring the name, doesn't need to be aliased

**Comment by barthuijgen** [comment]
Sorry, it's not supposed to have this much changes something went wrong in this branch. I'll try to fix it!

**Pull request closed without merging**